src/aqua/fluence/worker.aqua

```diff
@@ -32,6 +33,7 @@ func get_worker_settings(spell_id: SpellId) ->  ?WorkerSettings:
 
     settings: *WorkerSettings
 
+    -- TODO: I want to unite these guys to one structure but it won't be backward-compatible. 
```

Comment:
Backward-incompatible is fine, we clear state on Nox update anyway.

Reply:
Okay, I'll fix it then

But I remember I asked about it several times and the answer was that we didn't clear it and we preserved the previous state.